tests/trace_service.py

```diff
@@ -14,13 +14,15 @@ def Export(self, request, context):
         self.batches.append(request.resource_spans)
         return trace_service_pb2.ExportTracePartialSuccess()
 
-    def get_span(self):
+    def wait_one_batch(self):
         for _ in range(10):
             if len(self.batches):
                 break
             time.sleep(0.001)
-
         assert len(self.batches) == 1, "No spans received"
```

Comment:
This comment is unnecessary and somewhat confusing.

Reply:
Remove this comment.